### ISSUE TITLE
✨Docs to Meet

### ISSUE THREAD

**Issue by AntoLC** [numerique-gouv/impress#361]
## Feature Request

Could be nice to have a button on our doc to open in 1 click a visio meeting in Meet, it will increase the collaborative side of it.

With ID matching to share easily the meeting access ?
`https://docs.numerique.gouv.fr/docs/128e382a-cb8f-46fc-aa1a-b13dad4f3094/`
`https://visio.numerique.gouv.fr/128e382a-cb8f-46fc-aa1a-b13dad4f3094`

 # Thought

I guess this feature would be only working with La Suite environments.

**Comment by AntoLC** [comment]
@lebaudantoine Would it be easy to do ?

**Comment by virgile-dev** [comment]
Would be nice to have the same in Meet.
I'm in meeting, I decide to take notes, I can easily create a doc, from the meet interface. Maybe even edit it from the meet room.

**Comment by lebaudantoine** [comment]
Yes it's in the roadmap, but I first focus on the transcript+summary, it will create traffic in doc

**Comment by virgile-dev** [comment]
Since this feature has been in the backlog for a long time. It'd be nice to think of an implementation that doesn"t only work  in the DINUM context but for other actors that deploy meet and docs.

How about having an SDK that makes it easy to add :
- Create a doc for the meeting from the visio that is displayed in the Meet interface a bit like BBB did at the Hack Days ?
- If users don't want to create a doc cause they reuse always the same doc, propose a button "bring doc" which allows the user to look for a doc (maybe reusing the Drive file picker thing)

Sound like a cool summer project.

**Comment by lebaudantoine** [comment]
Actually, I would like to introduce kind of a pluging system, allowing to render any third party app in an iframe, instead of sharing my screen. 


> Create a doc for the meeting from the visio that is displayed in the Meet interface a bit like BBB did at the Hack Days ?

Should be feasible with the existing api /create-for-owner.

> If users don't want to create a doc cause they reuse always the same doc, propose a button "bring doc" which allows the user to look for a doc (maybe reusing the Drive file picker thing)

I would love to use the resource server, to get a view in the frontend of my docs and chose which one to open and share.


Also if we render a doc in an iframe in visio, it would be nice to have a specific view from docs, with the most minimalist interface possible.

**Comment by tibroc** [comment]
I think all of this would be nice, if it would **also** work with BigBlueButton. BBB has a Plugin-System already (as shown during the Hack Days) and integration would probably be easier if Docs hat some more API-Options that could be called from another server (similar to `/create-for-owner`). Is there are reason other endpoints can not be authorized through the server-to-server api tokens?

**Comment by rvveber** [comment]
Regarding plugins, for openDesk we have similar needs, therefore i have created a PluginSystem POC and Discussion 🙂
https://github.com/suitenumerique/docs/discussions/1242
